tools/data_converter/sunrgbd_data_utils.py

```diff
@@ -217,3 +217,107 @@ def process_single_scene(sample_idx):
         with futures.ThreadPoolExecutor(num_workers) as executor:
             infos = executor.map(process_single_scene, sample_id_list)
         return list(infos)
+
+
+class SUNRGBD_Calibration(object):
```

Comment:
We should implement this part by pytorch in box structures.

Reply:
Done. Also fixed some bugs or enhanced the API for point coordinate conversion